### DISCUSSION TITLE
Identity server in "not tirered" solution used for to connect from other applications

### DISCUSSION THREAD

**Discussion by crazygeek69** [abpframework/abp#8293] [General]
Hi,

Can we use an Identity Server in "not tiered" solution as authentication mechanism for other web applications, Clients...

Also for MVC tiered application always need to start IdendityServer, HTTPAPI.Host and Web projects. Is it the only way or easy way to run the web so that debug will be easy?

**Comment by amsaighi** [comment]
Yes, you can use Identity Server for other web application or clients ..., just configure others applications with your Identity server url .

**Reply by crazygeek69** [reply]
Thanks for replying back
